tests/jvm/src/test/scala/io/iteratee/files/TryTests.scala

```diff
@@ -0,0 +1,26 @@
+package io.iteratee.files
+
+import io.iteratee.{ Iteratee, Enumerator }
+import io.iteratee.tests.BaseSuite
+import scala.util.{ Success, Try }
+import java.io.File
+
+class TryTests extends BaseSuite {
```

Comment:
I just put it here to put the tests in the same package and path as the code. I don't know what this `it` means, but whatever the rule is, happy to move it.